BREAKING.md

```diff
@@ -40,24 +43,47 @@ This field has been deprecated and will be removed in a future breaking change.
 +       createShareLinkType?: ShareLinkTypes | ISharingLinkKind,
     ):
 ```
-
-
-### Remove `enableShareLinkWithCreate` from `HostStoragePolicy`
+### Remove enableShareLinkWithCreate from HostStoragePolicy
 `enableShareLinkWithCreate` feature gate has been deprecated and will be removed in a future breaking change. If you wish to enable creation of a sharing link along with the creation of Fluid file, you will need to provide `createShareLinkType:ISharingLinkKind` input to the `createOdspCreateContainerRequest` function and enable the feature using `enableSingleRequestForShareLinkWithCreate` in `HostStoragePolicy`
-### Add assertion that prevents sending op while processing another op
-`preventConcurrentOpSend` has been added and enabled by default. This will run an assertion that closes the container if attempting to send an op while processing another op. This is meant to prevent non-deterministic outcomes due to concurrent op processing.
-# 2.0.0
 
-## 2.0.0 Upcoming changes
-- [Remove `documentId` field from `MockFluidDataStoreContext`](#Remove-documentId-field-from-MockFluidDataStoreContext)
-- [Narrow type of `clientId` field on `MockFluidDataStoreRuntime`](#Narrow-type-of-clientId-field-on-MockFluidDataStoreRuntime)
-- [Remove `ConnectionState.Connecting`](#Remove-ConnectionState.Connecting)
-- [`IContainerRuntime.flush` is deprecated](#icontainerruntimeflush-is-deprecated)
+# 2.0.0-internal-1.0.0
+
+## 2.0.0-internal-1.0.0 Upcoming changes
```

Comment:
For future consideration: I'm doing some brainstorming about how the Upcoming changes are organized/presented.

1. Do we want the "upcoming" section to be tied to a version?
    * Pro of mentioning a version is it may be required to be on at least that version to get the alternative to migrate to.  If this is the common/typical case, then probably leave it.
    * Con is that this information gets spread throughout this file, which may make it harder to keep up with the list.  I'm also confused about if there'd be Upcoming sections for minor releases - looks like there are.

2. In light of this realization, maybe we want a separate file, so this info is not interleaved with true Breaking changes, especially since it's updated on a different cadence.  And I imagine there could also be weirdness keeping Upcoming change notes in sync between main and next, whereas that wouldn't so much be the case with Breaking (which would only be added to next, right?).

Reply:
cc @scottn12